packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -141,9 +141,13 @@ function SendConfirmation(props: Props) {
 
   const isInvite = !recipient.address
   const feeEstimates = useSelector(feeEstimatesSelector)
-  const feeEstimate = feeEstimates[tokenAddress]?.[isInvite ? FeeType.INVITE : FeeType.SEND]
+  const feeType = isInvite ? FeeType.INVITE : FeeType.SEND
+  const feeEstimate = feeEstimates[tokenAddress]?.[feeType]
 
   useEffect(() => {
+    if (!feeEstimate) {
```

Comment:
`feeEstimate` should be part of the effect dependencies.
And in a separate effect otherwise it may trigger the dispatch too many times.

Reply:
Moved it to its own `useEffect` since it could have been triggered more than once regardless by sharing the `useEffect` with the dek register estimation